### DISCUSSION TITLE
[Feature Request] Muti-language Contribution Support

### DISCUSSION THREAD

**Discussion by Chapoly1305** [seemoo-lab/AirGuard#160] [Ideas]
Hi there, 
great appreciation to your hard works on publication and developing this super cool anti-stalking application. I am very interested to promote this project to my friends and other people on the internet, however, this application seems to be limited in English or Deutsch only. 

I am a native Chinese speaker and have friends who can speak other languages wish to help to contribute this project by translating it. Wonder if there is a consideration to host a community translation project and open for contribution? I know website like https://crowdin.com (not endorsed, but highly recommended by another thousand starts project team) have such usage. 

Thank you for your consideration!

**Comment by yurtpage** [comment]
The app is translated to the Chinese itself but its app description is not.
See #225
